modules/end-user-guide/partials/proc_mounting-a-secret-as-an-environment-variable-into-a-workspace-container.adoc

```diff
@@ -1,23 +1,61 @@
 // Module included in the following assemblies:
-//
 // mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container
 
 [id="mounting-a-secret-as-an-environment-variable-into-a-workspace-container_{context}"]
 = Mounting a secret as an environment variable into a workspace container
 
-The following section describes how to mount a {platforms-name} secret from the user’s {orch-namespace} as an environment variable, or variables, into single-workspace or multiple-workspace containers of {prod-short}.
+In {prod-short}, you can mount the {platforms-name} secret from your {orch-namespace} as an environment variable (or variables) in single-workspace or multiple-workspace containers.
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+* A running instance of {prod-short}. See {link-installing-an-instance}.
 
 .Procedure
 
-. In the {platforms-namespace} where a {prod-short} workspace will be created, generate a new {platforms-name} secret.
-
-* The labels of the secret that is about to be generated must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. By default, it is a set of two labels: 
-+
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
+. Create a new secret in the {platforms-namespace} where a new workspace is deployed.
+. Define the default behavior for mounting secrets in the `automount-workspace-secret` annotation, which takes a boolean value:
+** `'true'` enables automatic mounting into all workspace containers.
+** `'false'` disables automatic mounting into all workspace containers. (To mount a secret, a user must request it in the devfile component by using the `automountWorkspaceSecrets:true` property.)
+. Set the name of the secret's environment variable in the `env-name` annotation.
+. Include the `mount-as: env` annotation.
+. Match the secret's `labels` to the set of labels configured in the `che.workspace.provision.secret.labels` property of {prod-short}.
+. Set the secret's key-value pair in the secret's `data` field.
+. If the secret must have multiple key-value pairs in the `data` field, set a different environment variable name with the `<__key__>_env-name` annotation for each key-value pair.
```

Comment:
Avoid 'if' unless necessary. Describe what to do. Don't introduce the need for weighing decisions.
```suggestion
. To set multiple key-value pairs in the `data` field of the secret, use a different environment variable name with the `__<key>___env-name` annotation format for each key-value pair.
```

Reply:
@rkratky I considered this sentence as a conditional, because the user does not have to complete this step unless the condition is met: "If the condition is met, then [you must must] do something." If we rewrite this as "To do something, do something," then we lose the conditional and expect the user to do something regardless of whether the condition is met or not. Here is my counter-proposal:

`. If you set multiple key-value pairs in the `data` field of the secret, then use a different environment variable name with the `__<key>___env-name` annotation format for each key-value pair.`